en/docs/use-cases/streaming-tutorials/triggering-integrations-via-micro-integrator.md

```diff
@@ -151,7 +158,34 @@ Let's design a Siddhi application that triggers an integration flow and deploy i
 
 7. Save the Siddhi application. As a result, it is saved in the `<SI_TOOLING_HOME>/wso2/server/deployment/workspace` directory.
 
-8. To deploy your Siddhi application, copy it from the `<SI_TOOLING_HOME>/wso2/server/deployment/workspace` directory, and then paste it in the `<SI_HOME>/WSO2/server/deployment/siddhi-files` directory.
+8. Click the **Deploy** menu option and then click **Deploy to Server**. The **Deploy Siddhi Apps to Server** dialog box opens as shown in the example below.
+
+    ![Deploy to Server dialog box]({{base_path}}/assets/img/streaming/getting-si-run-with-mi/deploy-to-server-dialog-box.png)
+
+    1. In the **Add New Server** section, enter information as follows:
+
+           | Field           | Value                            |
+           |-----------------|----------------------------------|
+           | **Host**        | Your host                        |
+           | **Port**        | `9443`                           |
+           | **User Name**   | `admin`                          |
+           | **Password**    | `admin`                          |
+
+        ![Add New Server]({{base_path}}/assets/img/streaming/getting-si-run-with-mi/add-new-server.png)
+
+        Then click **Add**.
+
+    2. Select the check boxes for the **grpc-call-response.siddhi** Siddhi application and the server you added as shown below.
+
+        ![Deploy Siddhi Apps to Server]({{base_path}}/assets/img/streaming/getting-si-run-with-mi/select-siddhi-app-and-server.png)
+
+    3. Click **Deploy**.
+
+        When the Siddhi application is successfully deployed, the following message appears in the **Deploy Siddhi Apps to Server** dialog box.
+
+        ![Deployment Status]({{base_path}}/assets/img/streaming/getting-si-run-with-mi/siddhi-application-deployment-status.png)
```

Comment:
When adding images, it is good to add the syntax in a way so that when the user clicks on the image the zoomed version of the image opens.